.github/workflows/docker_publish.yml

```diff
@@ -19,18 +19,17 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       # Emulate ARM Processor
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-      # Workaround: https://github.com/docker/build-push-action/issues/461
+        uses: docker/setup-qemu-action@v2
       - name: Setup Docker buildx
-        uses: docker/setup-buildx-action@79abd3f86f79a9d68a23c75a09a9a85889262adf
```

Comment:
🤔 was the tagging for the specific hash related to that issue commented above?

Reply:
I _think_ so. I had pulled most of this from a template, but I figured I'd give the latest versions a try as the issue is closed.